src/Radiance_Monitor/data_extract/ush/validate.sh

```diff
@@ -30,7 +30,6 @@ set -ax
    PDATE=$1
 
 
-   sat="sndrd4_g15"
```

Comment:
I wonder if `sat` should be initialized to an empty string in case `SATYPE_LIST` is an empty string.  Maybe just error/exit if that's the case?

Reply:
Good thoughts.  I've both initialized `sat `and included a test on `SATYPE_LIST`.  Thanks very much.